src/lib.rs

```diff
@@ -159,6 +261,128 @@ where
             channels: None,
         }
     }
+
+    pub async fn prerecorded_request(
+        &self,
+        source: impl AudioSource,
+        options: &OptionsBuilder<'_>,
```

Comment:
when I see types that end in `*Builder` they normally have a `.build()` method or similar which produces a `Options` type without the builder, you may want that `Options` type to be the parameter here, rather than the builder.

Reply:
Implemented in 0d9cc647e793b93cc3f8526002ecde41d62785bd.